product_code_builder_sequence/models/product_product.py

```diff
@@ -46,3 +46,18 @@ def copy(self, default=None):
                     'prefix_code': self.prefix_code + _('-copy'),
                 })
         return super(ProductTemplate, self).copy(default)
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.multi
+    def copy(self, default=None):
+        for product in self:
+            if default is None:
+                default = {}
+            if product.prefix_code:
```

Comment:
Or do you mean the same code for product.template, a couple of lines above? https://github.com/OCA/product-attribute/pull/182/files#diff-570b3649efcb95036f9fa9884a839eb2L44